docs/install.md

```diff
@@ -113,3 +113,49 @@ spanner each time follow the following steps:
     ```sh
     export SPANNER_EMULATOR_HOST=localhost:9010
     ```
+
+## Quickstart
```

Comment:
Wondering why this is labeled quick start and not Cloud Shell ?

Reply:
Quickstart because it is intended to a quick way to start using SMT. Cloud shell is the deployment platform for the quickstart.

This is similar to - https://cloud.google.com/spanner/docs/quickstart-console